tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot

```diff
@@ -243,6 +261,62 @@ Check data catalogue page contains archive and superseding publication subjects
     user checks page contains    This is the latest data
     user checks page does not contain    This is not the latest data
 
+Check data catalogue page contains superseded warning for archived publication (step one and two)
+    user navigates to data catalogue page on public frontend
+
+    user clicks radio    %{TEST_THEME_NAME}
+
+    user checks page contains element    //*[@data-testid="Radio item for ${PUBLICATION_NAME_ARCHIVE}"]
+    user checks page contains element    //*[@data-testid="Radio item for ${PUBLICATION_NAME_SUPERSEDE}"]
+
+    user clicks radio    ${PUBLICATION_NAME_ARCHIVE}
+    user clicks button    Next step
+    user waits for page to finish loading
+
+    user waits until page contains    Choose a release
+    user waits until page contains    ${RELEASE_NAME_ARCHIVE}
+
+    user checks page contains radio    ${RELEASE_NAME_ARCHIVE}
+    user checks page does not contain    This is the latest data
+    user checks page contains element    testid:superseded-warning
+    user checks page contains element    testid:superseded-by-link
+
+    # step 2
+    user clicks button    Next step
+    user checks page contains element    testid:superseded-warning
+    user checks page contains element    testid:superseded-by-link
+
+Check that superseded warning link takes user to superseding-publication data-catalogue step 1
+    user navigates to data catalogue page on public frontend
+
+    user clicks radio    %{TEST_THEME_NAME}
+
+    user checks page contains element    testid:Radio item for ${PUBLICATION_NAME_ARCHIVE}
+    user checks page contains element    testid:Radio item for ${PUBLICATION_NAME_SUPERSEDE}
+
+    user clicks radio    ${PUBLICATION_NAME_ARCHIVE}
+    user clicks button    Next step
+
+    user waits until page contains    Choose a release
+    user waits until page contains    ${RELEASE_NAME_ARCHIVE}
+
+    user checks page contains radio    ${RELEASE_NAME_ARCHIVE}
+
+    user checks page contains element    testid:superseded-warning
+    user checks page contains element    testid:superseded-by-link
+
+    # step 2
+    user clicks button    Next step
+
+    user checks page contains element    testid:superseded-warning
+    user checks page contains element    testid:superseded-by-link
+
+    user clicks element    testid:superseded-by-link
+
+    user waits until page contains    Browse our open data    %{WAIT_SMALL}
+
+    user waits until page contains    ${PUBLICATION_NAME_SUPERSEDE}    %{WAIT_SMALL}
```

Comment:
Might want to check that the page no longer contains the archived publication's name too